udfs/remote_udfs/README.md

```diff
@@ -0,0 +1,6 @@
+# Remote UDFs
+
+Remote UDFs in BigQuery utilize Cloud Functions or Cloud Run to run your custom code.  
+BigQuery connections are used to create an external connection to the Cloud Function or Cloud Run. 
```

Comment:
Can we remove the sentence on connection from this file?

Reply:
Removed